pkg/provisioner/ironic/ironic.go

```diff
@@ -731,6 +739,13 @@ func (p *ironicProvisioner) setDirectDeployUpdateOptsForNode(ironicNode *nodes.N
 	updater.
 		SetInstanceInfoOpts(optValues, ironicNode).
 		SetTopLevelOpt("deploy_interface", "direct", ironicNode.DeployInterface)
+
+	if p.config.liveISOForcePersistentBootDevice != "" {
+		driverOptValues := optionsData{
+			"force_persistent_boot_device": "Default",
```

Comment:
Shouldn't this be p.config.liveISOForcePersistentBootDevice instead of "Default"?

Reply:
No because this is in setDirectDeployUpdateOptsForNode, it is used in the normal ipi flow. The idea here is to set it to default regardless of whats configured so that the current ipi behaviour isn't changed.

Reply:
Then why condition it on `p.config.liveISOForcePersistentBootDevice != ""`? I think we should enforce the default value.